src/main/kotlin/org/wfanet/measurement/measurementconsumer/stats/Variances.kt

```diff
@@ -627,7 +627,7 @@ object VariancesImpl : Variances {
     return FrequencyVariances(
       relativeVariances = frequencyVariances.relativeVariances.mapValues { coefficient * it.value },
       kPlusRelativeVariances =
-        frequencyVariances.relativeVariances.mapValues { coefficient * it.value },
+        frequencyVariances.kPlusRelativeVariances.mapValues { coefficient * it.value },
```

Comment:
So this addressed the issue that we discussed yesterday -- the stds of rk and rk+ estimations are always equal. Am I right?